docs/development/codeguide.rst

```diff
@@ -165,17 +157,6 @@ Coding Style/Conventions
     need to be used in affiliated packages. In particular, the set of ruff_ checks is
     not required for affiliated packages.
 
-  * Alternately, you can manually check and fix your changes by running the
-    following `tox <https://tox.readthedocs.io/>`__ command::
-
-      tox -e codestyle
-
-* Following PEP8's recommendation, absolute imports are to be used in general.
-  The exception to this is relative imports of the form
-  ``from . import modname``, best when referring to files within the same
-  sub-module.  This makes it clearer what code is from the current submodule
-  as opposed to from another.
```

Comment:
This is enforced by Ruff rule [TID252 (relative-imports)](https://docs.astral.sh/ruff/rules/relative-imports/).